vpr/src/route/rr_graph_storage.cpp

```diff
@@ -505,7 +503,7 @@ void t_rr_graph_storage::partition_edges() {
     std::sort(
         edge_sort_iterator(this, 0),
         edge_sort_iterator(this, edge_src_node_.size()),
-        edge_compare_src_node_and_configurable_first(device_ctx.rr_switch_inf));
+        edge_compare_src_node_and_configurable_first(g_vpr_ctx.mutable_device().rr_graph_builder.rr_switch()));
```

Comment:
Add a comment here:

/* TODO: This API should be moved to ``RRGraphBuilder`` */